common/app/common/commercial/hosted/hardcoded/ZootropolisHostedPages.scala

```diff
@@ -84,6 +84,10 @@ object ZootropolisHostedPages {
     customData
   )
 
+
+  private lazy val videoPage = if (Switches.hostedArticle.isSwitchedOn) videoPageWithoutNextPage
```

Comment:
Might want to decouple article switch from video switch, or rename it if campaign can only launch in one go.